www/http-parser/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem                  1.0
+PortGroup                   github 1.0
+
+github.setup                nodejs http-parser 2.8.1 v
+categories                  www devel
+platforms                   darwin
+license                     MIT
+description                 http request/response parser for C.
+long_description            This is a parser for HTTP messages written in C. It parses both requests and responses. \
+                            The parser is designed to be used in performance HTTP applications. \
+                            It does not make any syscalls nor allocations, it does not buffer data, it can be interrupted at anytime. \
+                            Depending on your architecture, it only requires about 40 bytes of data per message stream (in a web server that is per connection).
+checksums                   rmd160  8ebe6c4b7ea28b40c31b1ef1abbfd7a89f57f7fb \
+                            sha256  6d6a66251dd4236c6e15565849777230f5bc78a92e16eb2d40cec1f7c0f5bbe3 \
+                            size    50744
+maintainers                 @Lord-Kamina
+patchfiles                  patch-Makefile.diff
+
+use_configure               no
+configure.cflags            -DHTTP_PARSER_STRICT=0 DESTDIR=${prefix}
```

Comment:
Is there a reason to set configure cflags if the configure phase is being skipped?

Reply:
Maybe I should change it to build.cflags I guess?

Reply:
Sure, if they are honored in the build phase.